hypha/apply/funds/views/partials.py

```diff
@@ -317,6 +320,13 @@ def sub_menu_update_status(request: HttpRequest) -> HttpResponse:
         else []
     )
 
+    # hide determination actions if submissions have different forms
+    if not allow_determination:
+        determination_actions = ["Dismiss", "Request More Information", "Accept"]
```

Comment:
@sandeepsajan0 Will this work with translations?

Reply:
@frjo It is fixed now, used machine names(DETERMINATION_OUTCOMES) to compare.